### PR TITLE
[Spending controls] Add allowance validation and add invite copy

### PR DESCRIPTION
**Pull request by malted**
`Yes` is the default now
image

Better copy
image

Also fixes alpine bug

### CODE REVIEW

File: app/views/organizer_position_invites/_organizer_position_invite.html.erb [hunk 1]
```diff
@@ -53,8 +53,10 @@
           </div>
         <% end %>
       </div>
-      <% if organizer_position_invite.initial_control_allowance_amount_cents.present? %>
-        (<%= render_money organizer_position_invite.initial_control_allowance_amount_cents %>)
+      <% if organizer_position_invite.initial_control_allowance_amount_cents.present? && !organizer_position_invite.initial_control_allowance_amount_cents.zero? %>
+        with a <%= render_money organizer_position_invite.initial_control_allowance_amount_cents %> allowance
+      <% else %>
+        with no allowance
```
Comment (sampoder) [inline app/views/organizer_position_invites/_organizer_position_invite.html.erb]: I think we can drop this

Comment (garyhtou) [reply app/views/organizer_position_invites/_organizer_position_invite.html.erb]: yeah, we should drop the "with no allowance". the "with a $1 allowance" is nice tho

File: db/schema.rb
Comment (garyhtou) [inline db/schema.rb:1]: drop the unrelated changes pls!

File: app/models/organizer_position/spending/control/allowance.rb [hunk 2]
```diff
@@ -41,6 +41,10 @@ def balance_is_positive
           errors.add(:control, "balance must be positive") if control.balance_cents + amount_cents < 0
         end
 
+        def balance_is_nonzero
```
Comment (sampoder) [inline app/models/organizer_position/spending/control/allowance.rb]:
```suggestion
        def balance_is_non_zero
```
Should this have a matching `validate`?

**Review by sampoder — APPROVED** [review]
LGTM!
